C++/include/algorithm/dynamic_programming/longest_increasing_subsequence.hpp

```diff
@@ -0,0 +1,44 @@
+
+/*
+  Longest Increasing Subsequence
+  ------------------------------
+  Given an unsorted array of integers, return the longest increasing subsequence length
+  Time complexity
+  --------------
+  O(N^2), where N is size of array
+  Space complexity
+  ---------------
+  O(N), where N is size of array
+  Author
+  ------
+  Shikhar Jaiswal(@shikherjaiswal)
+*/
+#ifndef longest_increasing_subsequence_hpp
+#define longest_increasing_subsequence_hpp
+#include <vector>
+int lis( std :: vector< int > arr, int n ) {
```

Comment:
Add a newline and function description above function. Look at codebase for formatting tips.